### PR TITLE
NO-ISSUE log OCP & mco version

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

File: internal/host/installcmd.go [hunk 1]
```diff
@@ -74,6 +74,7 @@ func (i *installCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models
 		return nil, err
 	}
 
+	i.log.Info("Install command releaseImage: %s, mcoImage: %s", releaseImage, mcoImage)
```
Comment (slaviered) [inline internal/host/installcmd.go:77]: I would recommend to log the entire command. It will help debugging too

Comment (oshercc) [reply internal/host/installcmd.go:77]: What do you mean by "entire command" ?

Comment (oshercc) [reply internal/host/installcmd.go:77]: /hold

Comment (slaviered) [reply internal/host/installcmd.go:77]: The command line in the following line, including the podman invocation

**Pull request merged**